server/src/ResolveLocation.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * kalman.c
```

Comment:
We started discussion on this earlier today. I think we should talk more on this.